_pages/en/strategy-open-first.md

```diff
@@ -0,0 +1,445 @@
+
+### Open First Center of Enablement (CoE)
+
+This strategy is firmly rooted in the ESDC's and the wider GC policy context.
+For it to succeed, a proper enabling environment must be put in place to guide, encourage and nudge change across the organisation.
+
+The strategy and the action plan are inextricably linked.
+The pivotal point will be an Open First CoE that enjoys genuine political and organisational support, involving all directorates-general.
+The CoE will act as facilitator for all activities outlined in the strategy and the action plan.
+It will assist subject-matter experts, encourage the directorates-general and help balance internal focus and external activities.
+
+As domain leader for IT, IITB will be responsible for setting up and running the Programme Office.
+This will allow it to set priorities, measure outcomes and report on progress.
+
+IITB will make an annual budget request for the CoE and the action plan, to be reviewed and approved through the normal budgetary procedures.
+Using the existing corporate IT governance structures, IITB will report on, monitor and review the action plan, and deliver a mid-term review of progress and implementation of the strategy.
+
+### Actions and impact
+
+The best way to lead is by example.
+This strategy will rapidly produce tangible evidence of the benefits of a working culture based on open principles.
+In addition, by simplifying the rules on sharing ESDC software and contributing to OSS projects, it will prove the value of open standards and OSS to the organisation and to society at large.
+
+| Actions | Impact |
+|---|---|
+| Set up Open First CoE | Expedite all activities in the action plan |
+| Set and promote the inner source default | Enable co-creation and collaboration |
+| Enhance the software repository | Enable co-creation and collaboration |
+| Revise software distribution practices | Process simplification; lower cost to society |
+| Enable and create innovation with open standards and OSS labs | Drive innovation; encourage co-creation |
+| Develop skills and recruit expertise | Contribute to staff recruitment and retention |
+| Increase outreach to communities | Encourage innovation; improve services |
+| Integrate open standards and OSS in internal IT governance | Increase organisational efficiency; foster innovation |
+| Ensure security | Increase value of open standards and OSS to the organisation and society |
+| Encourage and promote inner source | Instrumental to success of strategy and actions |
+
+These actions will provide the tools to reinforce and extend the open-source working culture.
+Our source code will gradually become available to all developer teams in ESDC and the Government of Canada.
+Once the legal barriers are removed, we will make an increasing number of our solutions publicly available.
```

Comment:
Do we have legal barriers?  This makes it sounds like something legal needs to change before the GC is able to release code as open source.  But I thought legally we were able to do this for code from the GC? You have to get the ok to do it but are there also legal barriers that prevent us from doing so?